packages/preset-mini/src/_variants/startingstyle.ts

```diff
@@ -0,0 +1,17 @@
+import type { Variant } from '@unocss/core'
+
+export const variantStartingStyle: Variant = {
+  name: 'start',
```

Comment:
I am not so sure about this variant; the name feels a bit too general and can be confusing. Since the feature is still experimental, I guess it might be better for you do it in-house at this moment.

Reply:
It is experimental but already in Chrome/Chrome for Android and Edge. https://caniuse.com/?search=%40starting-style
Webkit support coming soon: https://github.com/WebKit/WebKit/commit/1544606a8541c44e64d73ae4693c90da45ca6402

`start` could be `starting` instead (saw this in Tailwind, also not merged yet) but starting-style is what it's called basically.

No rush to merge (might as well wait for Tailwind PR review) but this will be useful for popovers using transitions instead of animations so could be nice to have if/when those become popular later.